src/ui/ListItem.tsx

```diff
@@ -0,0 +1,45 @@
+/** @license
+ *  Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+import * as React from "react";
+
+import { CSS } from "../lib/Constants";
+
+/**
+ * Interface for the properties assigned to the ListItem component.
+ * @interface
+ */
+interface IListItemProps {
+  title: string;
+  subtitle?: string;
+  inlineControl: boolean;
+  controlClass?: string;
+  control: any;
```

Comment:
It might be better to use `children` instead of `control`.  Then, your markup would look like this:

```javascript
<ListItem
  title = { title }
  subtitle = { title }
>
  <ColorSwatch />
</ListItem>
```

instead of

```javascript
<ListItem
  title = { title }
  subtitle = { title }
  control = {
    <ColorSwatch />
  }
/>
```

Ultimately, doesn't matter - it's just style.  It just looks a bit weird to see markup passed as a prop.